src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,69 @@
+package duke;
+
+import command.Command;
+import task.TaskList;
+import ui.UserInterface;
+
+import java.io.FileNotFoundException;
```

Comment:
The way you structure your imports is really clear and nicely ordered in alphabetical order.